app/javascript/consent-form.vue

```diff
@@ -389,9 +392,10 @@ export default {
     questionGroups() {
       return this.currentSurveyStep.groups
     },
-    checkboxAgreement() {
-      // TO DO: Change to include all question groups, not just the first
-      return this.answers[0].find(a => a.question_type == 'checkbox agreement')
+    checkboxAgreementQuestions() {
+      return this.answers.flatMap(
```

Comment:
Nice. For future reference `flatMap` calls just make me happy 🤷‍♂️ In fact, all functions that are just folds on iterables brighten my day 😉